apps/example/src/read-models/PayerReadModel.ts

```diff
@@ -1,38 +1,36 @@
-import { ITimelineEvent, StreamName } from "@equinox-js/core"
+import { StreamId } from "@equinox-js/core"
 import { Pool } from "pg"
 import { PayerId } from "../domain/identifiers.js"
 import { Payer } from "../domain/index.js"
-import { forEntity, Change, createProjection } from "@equinox-js/projection-pg"
+import { forEntity, Change, createHandler, MinimalClient } from "@equinox-js/projection-pg"
 
-type Payer = { id: PayerId; version: bigint; name: string; email: string }
+type Payer = { id: PayerId; name: string; email: string; version: string }
 
-const { Delete, Upsert } = forEntity<Payer, "id"| "version">()
+const { Delete, Upsert } = forEntity<Payer, "id" | "version">()
 
-export const projection = { table: "payer", id: ["id"], version: "version"}
+export const projection = { table: "payer", id: ["id"], version: "version" }
```

Comment:
projection vs project - good to remove overloading
any less ambiguous name?

Reply:
`projectionConfig`, `changes`?

Reply:
I think I prefer the former - changes works well as a name like you have in other places so better not to dilute that